lua/telekasten.lua

```diff
@@ -199,7 +199,7 @@ local function escape(s)
     return string.gsub(s, "[%%%]%^%-$().[*+?]", "%%%1")
 end
 
-local function make_absolute_path(path)
+local function make_config_path_absolute(path)
     local ret = path
     if not (Path:new(path):is_absolute()) and path ~= nil then
```

Comment:
future improvement: allow short-circuiting the and: test for `path ~= nil` before using path in `Path:new(path)`.

Reply:
Something like this ?

```lua
local function make_config_path_absolute(path)
    local ret = path
    if path ~=nil then
        if not(Path:new(path):is_absolute()) then
            ret = M.Cfg.home .. "/" .. path 
        end
   end
   return ret
end
```

Reply:
Oh, I just meant reversing the order of the `and`: 

```lua
if path ~= nil and not  Path:new....
```

But this will work fine, too

Reply:
aaaah yes, that makes sense.